package.json

```diff
@@ -89,7 +89,8 @@
     "ws": "^7.4.6",
     "trim-newlines": "^3.0.1",
     "normalize-url": "^4.5.1",
-    "path-parse": "^1.0.7"
+    "path-parse": "^1.0.7",
+    "set-value": "^4.0.1"
```

Comment:
should this update our `yarn.lock` too? did you run `yarn install`?

Reply:
It absolutely should, thanks for catching that! Pushed an update to make sure that that change is present.